runtime/doc/editing.txt

```diff
@@ -1455,7 +1455,7 @@ Do this before writing the file.  When reading an encrypted file it will be
 set automatically to the method used when that file was written.  You can
 change 'cryptmethod' before writing that file to change the method.
 
-To set the default method, used for new files, use this in your |vimrc| 
+To set the default method, used for new files, use this in your |vimrc|
```

Comment:
I would not remove trailing spaces as part of this change, it's causing many changes unrelated to unifying spelling of Vim.